Add release note for 20201104 (#30)
tidbcloud/dbaas-docs: release-notes.md

This page lists the release notes of [TiDB Cloud](https://pingcap.com/products/tidbcloud).

## November 4, 2020

> * Implement change organization name function

Reviewer: ```suggestion
* Implement the function of changing the organization name
```

> * Prevent users from accessing the TiDB under restoring

Reviewer: ```suggestion
* Prevent users from accessing TiDB during data restoring
```

> * Update Terms of Service and Privacy location in SigUp page

Reviewer: ```suggestion
* Update Terms of Service and Privacy location in the Sign Up page
```

> * Add feedback form entrance widget 

Reviewer: ```suggestion
* Add a feedback form entrance widget
```

> * Prevent Members from deleting owner in Preference tab

Reviewer: Plural or singular form? Please see which one is better.
```suggestion
* Prevent Members from deleting owner(s) in the Preference tab
```

> * Change TiFlash/TiKV storage chart metric

Reviewer: ```suggestion
* Change TiFlash and TiKV storage chart metrics
```

> * Upgrade default TiDB cluster to 4.0.8

Reviewer: ```suggestion
* Upgrade the default TiDB cluster version to 4.0.8
```
